WalletWasabi.Gui/ViewModels/ViewModelBase.cs

```diff
@@ -17,12 +17,14 @@ public class ViewModelBase : ReactiveObject, INotifyDataErrorInfo
 		public IEnumerable GetErrors(string propertyName)
 		{
 			var errorString = Validator.ValidateProperty(this, propertyName);
-			if (!string.IsNullOrEmpty(errorString))
+			if (string.IsNullOrEmpty(errorString))
+			{
+				return null;
+			}
+			else
```

Comment:
Don't need this else statement, could probably make this a ternary statement if we really wanted to

Reply:
What if the condition is false?

Reply:
If it is false, it will fail on the if statement and the go to the next line, which would be the return new List

Reply:
> Don't need this else statement

Your first comment is wrong. I simply inverted the if statement for clarity, there is no difference between the original code and this one.

Reply:
There was no else statement before..

Reply:
You can say that you prefer the original code.

> Don't need this else statement

You can't say that on the proposed code.